data/language/en-GB.txt

```diff
@@ -1861,6 +1861,7 @@ STR_2521    :Show staff list
 STR_2522    :Show recent messages
 STR_2523    :Show map
 STR_2524    :Screenshot
+STR_2525    :Giant Screenshot
```

Comment:
Always add new strings at the end. We don’t recycle IDs.

Reply:
Okay. Will push another commit.